backend/app/tests/storage_manager/storage/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from app.storage_manager.storage.storage import Storage
+
+
+@pytest.fixture(scope="class")
+def storage():
```

Comment:
You shouldn't need this. conftest fixtures are inherited down the module tree (but not up), so the one in `storage_manager/conftest.py` should be accessible here.